src/Console/Command/CompareCommand.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Command;
+
+use PhpCsFixer\Config;
+use PhpCsFixer\ConfigInterface;
+use PhpCsFixer\Console\ConfigurationResolver;
+use PhpCsFixer\FixerFactory;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Adamo Aerendir Crespi <hello@aerendir.me>
+ */
+final class CompareCommand extends Command
+{
+    const COMMAND_NAME = 'compare';
+
+    /**
+     * @var ConfigInterface
+     */
+    private $defaultConfig;
+
+    /**
+     * @var FixerFactory
+     */
+    private $fixerFactory;
+
+    /**
+     * @param null|FixerFactory $fixerFactory
+     */
+    public function __construct(FixerFactory $fixerFactory = null)
+    {
+        parent::__construct();
+
+        if (null === $fixerFactory) {
+            $fixerFactory = new FixerFactory();
+            $fixerFactory->registerBuiltInFixers();
+        }
+
+        $this->fixerFactory = $fixerFactory;
+        $this->defaultConfig = new Config();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDefinition(
+                [
+                    new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a .php_cs file.'),
+                    new InputOption('show-risky', '', InputOption::VALUE_REQUIRED, 'Shows also the riky fixers.', false),
+                    new InputOption('hide-configured', '', InputOption::VALUE_OPTIONAL, 'Hides all the rules currently configured to highlight only the ones not already in use.', false),
```

Comment:
It seems to me that both `show-risky` and `hide-configured` should be `InputOption::VALUE_NONE`.
Only if done this way, you can safely call them on boolean checks like
```php
if ($input->getOption('hide-configured')) { /* ... */ }
```
instead of
```php
$hideConfigured = null === $input->getOption('hide-configured') ? true : false;
```

Reply:
Good advice @Slamdunk ... Done!